module/dialogs/FightDialog.ts

```diff
@@ -111,19 +113,17 @@ export class FightDialog extends ExtendedTestDialog<FightDialogData> {
             this._toggleHidden(e.target);
         });
         html.find('.fighters-grid input, .fighters-grid select').on('change', (e: JQuery.ChangeEvent) => this.updateCollection(e, this.data.data.participants));
-
-        html.find('button[data-action="rollSpeed"]').on('click', (e: JQuery.ClickEvent) => { this._handleRoll(e, "speed"); });
-        html.find('button[data-action="rollPower"]').on('click', (e: JQuery.ClickEvent) => { this._handleRoll(e, "power"); });
-        html.find('button[data-action="rollAgility"]').on('click', (e: JQuery.ClickEvent) => { this._handleRoll(e, "agility"); });
-        html.find('button[data-action="rollSkill"]').on('click', (e: JQuery.ClickEvent) => { this._handleRoll(e, "skill"); });
-
-        html.find('img[data-action="openSheet"]').on('click', (e: JQuery.ClickEvent) => {
-            const id = e.target.dataset.actorId || "";
+        ["Speed", "Power", "Agility", "Skill", "Steel"].forEach((attr: string) => {
+            html.find('button[data-action="roll'+attr+'"]')
```

Comment:
```suggestion
            html.find(`button[data-action="roll${attr}"]`)
```
Prefer string interpolation here